enginetest/queries/charset_collation_engine.go

```diff
@@ -313,7 +313,7 @@ var CharsetCollationEngineTests = []CharsetCollationEngineTest{
 			{
 				Query: "SHOW CREATE TABLE test4;",
 				Expected: []sql.Row{
-					{"test4", "CREATE TABLE `test4` (\n  `pk` bigint NOT NULL,\n  `v1` varchar(255) COLLATE utf8mb4_unicode_ci,\n  PRIMARY KEY (`pk`)\n) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_bin"},
+					{"test4", "CREATE TABLE `test4` (\n  `pk` bigint NOT NULL,\n  `v1` varchar(255) COLLATE utf8mb4_unicode_ci\n) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_bin"},
```

Comment:
I had to double check this one, but it looks like my test was wrong! It doesn't carry over primary keys, which makes sense as it also doesn't carry over indexes, so it would have to special-case it for primary keys, which doesn't really make sense. A small detail that I didn't catch, just assumed it brought those over. I checked this against MySQL the first time too, that makes it even worse lol

Reply:
Also I saw the comment later on regarding stripping the schema. Wrote the above comment before I got to that part in the review.

Reply:
The rare Daylon error, hard to catch you slippin